test/test.js

```diff
@@ -233,14 +233,23 @@ test('v-stream directive (with .stop, .prevent modify)', done => {
 })
 
 test('v-stream directive (with data)', done => {
+  const customButton = {
+    name: 'custom-button',
+    template: `<button id="custom-button" @click="$emit('click')"><slot/></button>`
```

Comment:
Use ‘klick’ or other event name to reduce variables and prevent potential misdetection in test cases

Reply:
@regou I've done the aforementioned feedback. Thanks.